### PR TITLE
HA Fixes

### PR DESCRIPTION
**Pull request by Stingray540**
Adds fix for step-move and AI spawns

### CODE REVIEW

**Review by BDpuffy420 — CHANGES_REQUESTED** [review]
Remove the admin rank/admin.txt

**Review by Stingray540 — COMMENTED** [review]
They're the stock files, and travis really doesn't like it when they are missing.

**Review by BDpuffy420 — APPROVED** [review]

**Comment by BDpuffy420** [comment]
Fuck you @travis-ci

File: maps/geminus_city/geminus_city_jobs.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 /datum/map/geminus_city
 	allowed_jobs = list(/datum/job/colonist,/datum/job/innie_sympathiser,/datum/job/insurrectionist_recruiter,/datum/job/colonist_mayor,/datum/job/marine,/datum/job/marine_leader,/datum/job/police,/datum/job/cop)
-	allowed_spawns = list("Colony Arrival Shuttle","UNSC Peacekeeping Ship","UNSC Peacekeeping Ship - Leader Quarters")
+//	allowed_spawns = list("Colony Arrival Shuttle","UNSC Peacekeeping Ship","UNSC Peacekeeping Ship - Leader Quarters")
```
Comment (caelaislinn) [inline maps/geminus_city/geminus_city_jobs.dm:4]: This should be `allowed_spawns = list(DEFAULT, "Colony Arrival Shuttle","UNSC Peacekeeping Ship","UNSC Peacekeeping Ship - Leader Quarters")

File: config/example/admin_ranks.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+GameAdmin       +@ +DEBUG +FUN +POSSESS +BUILDMODE +SOUND +PERMISSIONS
+SeniorAdmin     +@
+
+HeadDeveloper   +EVERYTHING
+HeadAdmin       +EVERYTHING
+Host            +EVERYTHING
+
+Developer       +DEBUG +VAREDIT +SERVER +SPAWN +REJUV +POSSESS +BUILDMODE +ADMIN +FUN
+TrialDevAdmin   +DEBUG +VAREDIT +SERVER +SPAWN +REJUV +POSSESS +BUILDMODE +ADMIN +STEALTH +BAN +FUN
+DevAdmin        +DEBUG +VAREDIT +SERVER +SPAWN +REJUV +POSSESS +BUILDMODE +ADMIN +STEALTH +BAN +FUN +SOUND +PERMISSIONS
```
Comment (caelaislinn) [inline config/example/admin_ranks.txt:46]: Never delete example files hehe

File: maps/geminus_city/geminuscity_2.dmm [hunk 3]
```diff
@@ -300,6 +300,7 @@
 "afN" = (/obj/effect/landmark/start{name = "Shaft Miner"},/turf/simulated/floor/decor{color = "#cccccc"; icon_state = "square_grey"; tag = "icon-tiled1"},/area/planets/Geminus/outdoor/north)
 "afO" = (/obj/structure/window/reinforced,/obj/structure/closet/secure_closet/security,/turf/simulated/floor/tiled,/area/planets/Geminus/indoor/police_station)
 "afP" = (/obj/machinery/door/airlock/diamond{name = "Bathroom"},/turf/simulated/floor/tiled,/area/planets/Geminus/outdoor/north)
+"afQ" = (/obj/effect/landmark/start/AI,/turf/simulated/floor/greengrid,/area/planets/Geminus/indoor/science)
```
Comment (caelaislinn) [inline maps/geminus_city/geminuscity_2.dmm:303]: Geminus bugs reappearing suggests a problem somewhere else. Geminus isn't live so don't worry about trying to fix these issues... I'll dig into them later today

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: maps/geminus_city/geminus_city_jobs.dm (hunk 1 above)
Comment (BDpuffy420) [reply maps/geminus_city/geminus_city_jobs.dm:4]: We can fix it later

**Pull request merged**